### PR TITLE
BC-6410 - expose minIO console on a fixed port number

### PR DESCRIPTION
**Pull request by bn-pass**
# Description

The purpose of this ticket is to expose the minIO GUI console on a fixed port number and also expose it on its k8s Service.

## Links to Tickets or other pull requests

https://ticketsystem.dbildungscloud.de/browse/BC-6410

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by mamutmk5 — APPROVED** [review]

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
Miss the removing of the variable at ansible/group_vars/production/storage.yml

**Review by mamutmk5 — APPROVED** [review]

File: ansible/group_vars/reference/storage.yml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 STORAGE_IMAGE_NAME: docker.io/minio/minio:RELEASE.2022-10-24T18-35-07Z
```
Comment (virgilchiriac) [inline ansible/group_vars/reference/storage.yml:1]: @mamutmk5 we don't use minio in production, do we?

File: ansible/group_vars/production/storage.yml [hunk 2]
```diff
@@ -1,3 +1,2 @@
 STORAGE_IMAGE_NAME: docker.io/minio/minio:RELEASE.2022-10-24T18-35-07Z
```
Comment (virgilchiriac) [inline ansible/group_vars/production/storage.yml:1]: @mamutmk5 we don't use minio in production, do we?

Comment (mamutmk5) [reply ansible/group_vars/production/storage.yml:1]: no, but is better to set if we need it.

**Pull request merged**